Apache-Beam-Jobs.md

```diff
@@ -691,6 +692,32 @@ some_values = (
     | 'Get values' >> beam.Values()
 )
 ```
+## Guidelines for writing beam jobs
+This section provides some general guidelines for writing Beam jobs, which would be particularly helpful for new contributors.
+
+### Planning a job
+* If your beam job includes updating the storage models, make sure to write an audit job. An audit job is similar to your beam job but doesn't make any changes to the datastore. During testing, the audit job is run before the actual beam job to prevent any unwanted changes to the datastore incase of an error. You will be able to find examples in the codebase for the same.
+* There should also be a job to verify the changes done by your job.
```

Comment:
Done

Reply:
The bullet point here still reads as though it's standalone. Maybe put the previous point below it so that it's clear that the previous point is an example of this point? Otherwise, it looks like the previous point is an example of the point before it.

Reply:
@seanlip Do you mean to say I should interchange the points "If your beam job..." and "Consider the example..."?

Reply:
Ah, I just mean that the actual example should be after the point that it is supposed to be an example for. If it is an example for line 703 then the line with the example should come after line 703. Does that help clarify?

Reply:
Yes it does. Thanks for the clarification @seanlip.